WARNING:root:Could not parse http://www2.assemblee-nationale.fr/recherche/resultats_questions after 10 tries
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))': /session/b2c6c739-0fc7-4dd7-841d-b1a144965970
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f44f52fc860>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/b2c6c739-0fc7-4dd7-841d-b1a144965970
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f44f52fcac8>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/b2c6c739-0fc7-4dd7-841d-b1a144965970
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8764cfcac8>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/5dfc9a71-9a78-4fa9-8af0-c17309a9c7cf
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8764ca8208>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/5dfc9a71-9a78-4fa9-8af0-c17309a9c7cf
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8764323ef0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/5dfc9a71-9a78-4fa9-8af0-c17309a9c7cf
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fb531326c18>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/8b3e97e6-03b2-4bcc-988b-b2e5d0c4cd4b
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fb5312d1278>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/8b3e97e6-03b2-4bcc-988b-b2e5d0c4cd4b
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fb530935ac8>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/8b3e97e6-03b2-4bcc-988b-b2e5d0c4cd4b
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5bcf1e9c88>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/ab4c27eb-d009-4b21-b4bf-8dc6fc018aee
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5bcf1832e8>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/ab4c27eb-d009-4b21-b4bf-8dc6fc018aee
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5bce7f7b38>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/ab4c27eb-d009-4b21-b4bf-8dc6fc018aee
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))': /session/a9e09193-fcca-4037-8dc4-4594292d9856
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fedabf991d0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/a9e09193-fcca-4037-8dc4-4594292d9856
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fedab5fbb00>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/a9e09193-fcca-4037-8dc4-4594292d9856
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fd8f8528cc0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/0eec3d59-6fe9-4148-8c2c-19c94fd033f9
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fd8f7a8d2e8>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/0eec3d59-6fe9-4148-8c2c-19c94fd033f9
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fd8f7a8deb8>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/0eec3d59-6fe9-4148-8c2c-19c94fd033f9
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ffab2ebab38>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/2f776a3f-d1e6-4753-9981-2685831bd6b3
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ffab2e69240>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/2f776a3f-d1e6-4753-9981-2685831bd6b3
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7ffab24ccb38>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/2f776a3f-d1e6-4753-9981-2685831bd6b3
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f06b2ff9c18>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/12aface0-2d4c-4fd4-8c54-b3811f44ce9c
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f06b2f9d2b0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/12aface0-2d4c-4fd4-8c54-b3811f44ce9c
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f06b260cb38>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/12aface0-2d4c-4fd4-8c54-b3811f44ce9c
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /session/55be03b0-4095-4505-a21c-5fff32114445
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f83c68bf240>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/55be03b0-4095-4505-a21c-5fff32114445
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f83c5f25b38>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/55be03b0-4095-4505-a21c-5fff32114445
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f2f828cdc50>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/557b488b-a907-444e-930e-b4b5de1f39a4
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f2f81e8d860>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/557b488b-a907-444e-930e-b4b5de1f39a4
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f2f81e8d828>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/557b488b-a907-444e-930e-b4b5de1f39a4
